standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -1979,7 +1979,7 @@ struct GetPartitionsRequest {
    4: optional bool withAuth,
    5: optional string user,
    6: optional list<string> groupNames,
-   7: GetPartitionsProjectionSpec projectionSpec
+   7: GetProjectionsSpec projectionSpec
    8: GetPartitionsFilterSpec filterSpec, // TODO not yet implemented. Must be present but ignored
```

Comment:
Not related to your change, but looks like that TODO should be removed since we have the filterspec implemented.

Reply:
I generally thought it is forbidden to make changes unrelated to the JIRA in a patch. Ofcourse this is a minor change and I will do as you suggest. Thank you for the comment.